scripts/Schemas.scala

```diff
@@ -43,8 +43,18 @@ private def fetchSchemas(cwd: os.Path): Unit =
   val targetPath = cwd / "integration-tests" / "resources" / "testdata"
   os.remove.all(targetPath)
 
+  // copy test schemas
   copySchemas(pulumiRepo / "pkg" / "codegen" / "testing" / "test" / "testdata", targetPath)
   copySchemas(pulumiJavaRepo / "pkg" / "codegen" / "testing" / "test" / "testdata", targetPath)
+
+  // fetch sample production schemas
+  List(
+    "kubernetes",
```

Comment:
There are already some schemas (including these providers) in pulumi repo testdata. Why don't we use these?